s3access/select_spec.py

```diff
@@ -0,0 +1,232 @@
+import dataclasses
+from datetime import date, datetime
+from enum import Enum
+from typing import Dict, Optional, Union, Sequence, Tuple, Any
+
+
+# https://docs.aws.amazon.com/AmazonS3/latest/API/API_SelectObjectContent.html
+
+
+class ExpressionType(Enum):
+    SQL = 'SQL'
+
+
+class InputSerialization(Enum):
```

Comment:
I am a bit undecided on these ones.

With the advent of `typing` I for one became a huge fan of `Literal['Foo', 'Bar']` which I think makes definitions much more succinct and feels more pythonic than those javaesque Enums.

Reply:
I don't mind changing these to literals, though i find auto-completion to be easier in this way.

Reply:
I do not mind either way.